src/Schemes/CoveredSchemes/Objects/Attributes.jl

```diff
@@ -157,13 +157,36 @@ end
   return domain(inc), inc
 end
 
-@attr function singular_locus(X::AbsCoveredScheme)
+@attr function singular_locus(
+    X::AbsCoveredScheme;
+  )
```

Comment:
This line break seems non-standard. To me it just makes the code harder to parse.